evalml/model_understanding/graphs.py

```diff
@@ -890,3 +891,59 @@ def graph_prediction_vs_actual_over_time(pipeline, X, y, dates):
                         yaxis={'title': 'Target Values and Predictions'})
 
     return _go.Figure(data=data, layout=layout)
+
+
+def t_sne(X, n_components=2, perplexity=30.0, learning_rate=200.0, metric='euclidean'):
+    """Get the transformed output after fitting X to the embedded space.
+
+     Arguments:
+        X (ww.DataTable, pd.DataFrame): Features.
```

Comment:
If you accept ww.DataTable as X input, you should convert `to_dataframe()` before you call TSNE, otherwise it throws an error. 
![image](https://user-images.githubusercontent.com/22552445/105865722-322bf300-5fc1-11eb-99c2-ea42eb7d4735.png)